tensorboardX/summary.py

```diff
@@ -123,13 +150,13 @@ def make_histogram(values, bins):
     # void Histogram::EncodeToProto in histogram.cc
     for i, c in enumerate(counts):
         if c > 0:
+            start = i - 1
```

Comment:
What does it fix? btw, it should be in a separated commit

Reply:
It was a style fix - when it was outside the loop it was complaining that the variable i wasn't being used and should be replaced by a _.